Tests/XCUITests/ExperimentIntegrationTests.swift

```diff
@@ -0,0 +1,67 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+
+final class ExperimentIntegrationTests: BaseTestCase {
+
+    override func setUpApp() {
+        super.setUpApp()
+        app.activate()
+        let closeButton = app.buttons["bottomSheet close"]
+        if closeButton.exists {
+            closeButton.tap()
+        }
+        super.setUpScreenGraph()
+    }
+
+    override func tearDownWithError() throws {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+    }
+
+    func testVerifyExperimentEnrolled() throws {
+        let toolbarClosePredicate = NSPredicate(format: "label CONTAINS 'Close'")
+        let predicate = NSPredicate(format: "identifier CONTAINS 'FxVersion'")
+        let toolbarClose = app.buttons.matching(toolbarClosePredicate)
+
+        toolbarClose.element(boundBy: 0).tap()
+        navigator.goto(SettingsScreen)
+
+        // enable experiments secret menu
+        let element = app.tables.cells.containing(predicate)
+        for _ in 0...5 {
+            element.element.tap()
+        }
+
+        // open experiments menu
+        let experiments = app.tables.cells.containing(NSPredicate(format: "label CONTAINS 'Experiments'"))
+        experiments.element.tap()
+
+        // match json experiment name
+        let jsonString = ProcessInfo.processInfo.environment["EXP_NAME"]!
+        let experiment = app.tables.cells.containing(NSPredicate(format: "label CONTAINS[c] '\(jsonString)'"))
+        if experiment.count > 0 {
```

Comment:
Swiftlint failed due to ```Prefer checking `isEmpty` over comparing `count` to zero```
```suggestion
        if !experiment.isEmpty {
```